src/server/api/routers/analytics.router.ts

```
@@ -84,4 +105,4 @@ export const analyticsRouter = createTRPCRouter({
     .query(async ({ ctx }) => {
       return analyticsService.getTrends(ctx.db);
     }),
-});
+});
```

ESLint

[error] Insert `⏎`

(prettier/prettier)